src/encoded/static/components/regulome_summary.js

```diff
@@ -48,17 +48,17 @@ const snpsColumns = {
 const SNPSummary = (props) => {
     const snps = props.context.summaries;
     return (
-        <div>
+        <React.Fragment>
```

Comment:
Doesn’t look like the old `<div>` was necessary in the first place. I think `<SortTablePanel>` could be the outer component here.